src/components/Slider.tsx

```diff
@@ -146,42 +52,89 @@ const Slider: React.FC<IProps> = ({ children, onSlideChange }) => {
   }, [embla, setScrollSnaps, onSelect])
 
   return (
-    <Embla>
-      <EmblaViewport ref={emblaRef}>
-        <EmblaContainer>{children}</EmblaContainer>
-      </EmblaViewport>
-      <Buttons>
-        <Button onClick={scrollPrev} disabled={!prevBtnEnabled}>
-          <Icon
-            name="arrowLeftIos"
-            color={
-              prevBtnEnabled
-                ? theme.colors.slider.btnColor
-                : theme.colors.slider.btnColorDisabled
-            }
-          />
-        </Button>
-        <Button onClick={scrollNext} disabled={!nextBtnEnabled}>
-          <Icon
-            name="arrowRightIos"
-            color={
-              nextBtnEnabled
-                ? theme.colors.slider.btnColor
-                : theme.colors.slider.btnColorDisabled
-            }
-          />
-        </Button>
-      </Buttons>
-      <Dots>
+    <Box
+      position="relative"
+      p="2rem"
+      borderWidth="1px"
+      borderStyle="solid"
+      borderRadius="0.3rem"
+      w="full"
+      bg="sliderBg"
+    >
+      <Box overflow="hidden" ref={emblaRef}>
+        <Flex>{children}</Flex>
+      </Box>
+      <Flex
```

Comment:
I think we should position this as `absolute` as before. If not it looks weird.

![image](https://user-images.githubusercontent.com/468158/197211060-d8bd3310-5a57-4de2-8d68-8e2fd1ceecab.png)

The dots should be centered with the buttons. Take a look at [the prev implementation](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Slider.tsx#L52).

Reply:
Sure the buttons looks odd and I did try the prev implementation but there was some issues with the alignment so I made it in a presentable way.

Also as you said the button should be aligned center.
I pushed some changes that should fix it

## Attaching screen shots for new and old changes
### New Pushed changes as this comment date
<img width="479" alt="Screenshot 2022-10-22 at 6 26 32 PM" src="https://user-images.githubusercontent.com/52166437/197340417-79ab1562-ec30-407f-a2a1-447112922136.png">

### Old changes as in the live [website](https://ethereum.org/en/what-is-ethereum/)
<img width="838" alt="Screenshot 2022-10-22 at 5 13 01 PM" src="https://user-images.githubusercontent.com/52166437/197340450-715ad3b8-9747-4b1f-bf69-7425bc2aad76.png">

Reply:
Also I made sure it doesn't break for mobile view